From 17d961d7b14dc322507f8b0d1fc67d0f658ccdf5 Mon Sep 17 00:00:00 2001
From: Colin Walters <walters@verbum.org>
Date: Thu, 24 Jan 2013 17:47:45 -0500
Subject: [PATCH] selinux-access: Delete debugging message logged as an error

I don't see why this should be logged at all, so let's delete it.
(cherry picked from commit 6fd55349ab1c392b01d7979f115f8be2cd89d76a)
---
 src/core/selinux-access.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/src/core/selinux-access.c b/src/core/selinux-access.c
index b933698..41f824a 100644
--- a/src/core/selinux-access.c
+++ b/src/core/selinux-access.c
@@ -308,8 +308,6 @@ static int get_calling_context(
         */
         sender = dbus_message_get_sender(message);
         if (sender) {
-                log_error("SELinux Got Sender %s", sender);
-
                 r = bus_get_selinux_security_context(connection, sender, scon, error);
                 if (r >= 0)
                         return r;
